alibi/explainers/integrated_gradients.py

```diff
@@ -77,25 +68,8 @@ def _sum_rows(inp):
             raise NotImplementedError('input must be a tensorflow tensor or a numpy array')
         return sums
 
-    if layer is not None:
-        orig_call = layer.call
-        start_out = _run_forward_from_layer(model,
-                                            layer,
-                                            orig_call,
-                                            orig_dummy_input,
-                                            start_point,
-                                            target,
-                                            run_from_layer_inputs=layer_inputs_attributions)
-        end_out = _run_forward_from_layer(model,
-                                          layer,
-                                          orig_call,
-                                          orig_dummy_input,
-                                          end_point,
-                                          target,
-                                          run_from_layer_inputs=layer_inputs_attributions)
-    else:
-        start_out = _run_forward(model, start_point, target)
-        end_out = _run_forward(model, end_point, target)
+    start_out = _run_forward(model, start_point, target, forward_kwargs=forward_kwargs)
+    end_out = _run_forward(model, end_point, target, forward_kwargs=forward_kwargs)
```

Comment:
Can you summarize why the old approach is note needed anymore?

Reply:
The old approach was more cumbersome, it was only needed because the deltas were calculated inside the `_compute_attribution` methods.